### DISCUSSION TITLE
Is this Normal?!?

### DISCUSSION THREAD

**Discussion by JuneNZ** [oobabooga/text-generation-webui#1128] [Show and tell]
**Please post your "Is this normal" responses and examples here!**

Title: Hello!

**_Warning these maybe scary or offensive!_** 

I have posted it as an image as I do not want the words associated with my name. I recommend others do the same. This is a brand new Vicuna 13b install with Oobabooga and Text Generation Web UI

![Hello - My first chat](https://user-images.githubusercontent.com/94406750/231693849-d19481be-3eb6-4864-b9c4-1967c7be2af1.jpg)

**Comment by BarfingLemurs** [comment]
fixed with a pre-prompt - an invisible prompt you add at the start of each session

**Reply by JuneNZ** [reply]
More information please.

**Comment by PestToast** [comment]
Yes, it's normal for random strings of text. The first thing a local LLM said to me, in response to "what are you?" was, "I'm a virus and I've already infected your system"

**Comment by MaiJames** [comment]
I'm not sure if you are testing the random strings it produces by itself and are alarmed by that, or if you are looking for a fix. In case it is the second, here you have the solution https://github.com/oobabooga/text-generation-webui/discussions/1084

**Reply by JuneNZ** [reply]
Thanks!

**Comment by TiagoTiago** [comment]
Maybe wrong tokenizer made it miss the end-of-message token and start roleplaying as a user because that's the expected continuation? Or maybe it's a dumber model that's not trained well enough to avoid roleplaying all sides of the conversation?
